src/pairwise.jl

```diff
@@ -134,7 +145,7 @@ end
 
 # Identical to `Base.promote_typejoin` except that it uses `promote_type`
 # instead of `typejoin` to combine members of `Union` types
-function promote_type_union(::Type{T}) where T
+function promote_type_union(::Type{T}) where {T}
```

Comment:
```suggestion
function promote_type_union(::Type{T}) where T
```

Reply:
Again this is VS Code Format Document doing its stuff.